Is "A003 class attribute `set` is shadowing a python builtin" too aggressive?

Dear Charlie,

after upgrading to `ruff 0.0.52` on https://github.com/isarengineering/SecPi, we found `ruff` would emit an admonition on a code like outlined below. Do you think it might be worth to build up a dedicated exclusion list for specific symbols, in order make ruff skip those cases?

With kind regards,
Andreas.

```python
class Foobar:
    def __init__(self):
        self.data = {}

    def get(self, key, default=None):
        return self.data.get(key, default)

    def set(self, key, value):
        self.data[key] = value
```
```
$ ruff foobar.py
Found 1 error(s).
foobar.py:8:5: A003 class attribute `set` is shadowing a python builtin
```

P.S.: The new features and rules added to ruff keep being an awesome driver for modernizing an outdated Python 2 code base, where I did not bother to run flake8 on it at all, see https://github.com/isarengineering/SecPi/pull/30. Cheers!

--- a/src/checks.rs
+++ b/src/checks.rs
@@ -5,7 +5,7 @@ use anyhow::Result;
 use rustpython_parser::ast::Location;
 use serde::{Deserialize, Serialize};
 
-pub const DEFAULT_CHECK_CODES: [CheckCode; 48] = [
+pub const DEFAULT_CHECK_CODES: [CheckCode; 42] = [
     // pycodestyle
     CheckCode::E402,
     CheckCode::E501,
@@ -50,15 +50,6 @@ pub const DEFAULT_CHECK_CODES: [CheckCode; 48] = [
     CheckCode::F831,
     CheckCode::F841,
     CheckCode::F901,
-    // flake8-builtins
-    CheckCode::A001,
-    CheckCode::A002,
-    CheckCode::A003,
-    // flake8-print
-    CheckCode::T201,
-    CheckCode::T203,
-    // flake8-super
-    CheckCode::SPR001,
 ];
 
 pub const ALL_CHECK_CODES: [CheckCode; 51] = [

